udata/features/identicon/backends.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import hashlib
+import io
+import sys
+
+import pydenticon
+
+from flask import redirect, send_file, current_app
+
+from udata import theme, entrypoints
+
+ADORABLE_AVATARS_URL = 'https://api.adorable.io/avatars/{size}/{identifier}.png'
+ROBOHASH_URL = 'https://robohash.org/{identifier}.png?size={size}x{size}&set={skin}&bgset={bg}'  # noqa
+
+
+# Default values overriden by theme default and local config
+DEFAULTS = {
```

Comment:
Why aren't the defaults stored in `settings.py`?

Reply:
To let the theme bring custome parameters.